.github/workflows/python-whl-osx.yaml

```diff
@@ -0,0 +1,56 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: Python 37 wheel osx
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+
+  build-wheel:
+    name: build-wheel
+    runs-on: macos-latest
```

Comment:
we need to generate python clients for multiple osx versions, not just the latest mac. How can we do that?

Reply:
github only supports osx 10.15, one option is to use azure pipelines which has osx10.14 also but better option would be to change the setup.py and cmake files to build for older versions of osx on 10.15, it's doable requires setting a few environments variables and custom linker flags.